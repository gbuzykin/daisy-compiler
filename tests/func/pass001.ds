namespace std {

func f(x: i32, y: i32) -> i32;

func f(x: i32, y: i32) -> i32;

func f(x: i32, y: i32) -> i32 {}

func f(&mut x: i32, y: i32) -> i32 {}

}
