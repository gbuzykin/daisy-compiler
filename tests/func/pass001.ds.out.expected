./func/pass001.ds: info: warnings 0, errors 0
